conversion/conversion.go

```diff
@@ -299,6 +332,9 @@ func dataFromDatabase(ctx context.Context, sourceProfile profiles.SourceProfile,
 			}
 			return bw, nil
 		}
+		if sourceProfile.Conn.Dataproc {
+			return performDataprocMigration(sourceProfile, targetProfile, config, conv, client, infoSchema)
+		}
```

Comment:
I don't think this is the right integration point, we should had a new config Type to `SourceProfileTypeConfig`.

This will make the integration much more easier and readable. Add a new type for Dataproc similar to the existing ones here - 
https://github.com/cloudspannerecosystem/harbourbridge/blob/master/profiles/source_profile.go#L486

Reply:
I am not sure how we are ensuring that this works only for `MySQL` source. Can you please validate?

Reply:
Added new `SourceProfileTypeConfig ` for storing dataproc config
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/520/commits/d65e9ff652f5c9adf3145b8c9c860c02076a8ad6